lib/thrift/binary/framed/protocol_handler.ex

```diff
@@ -54,7 +69,8 @@ defmodule Thrift.Binary.Framed.ProtocolHandler do
         :ok = transport.close(socket)
 
       {:error, reason} ->
-        Logger.info(fn -> "#{inspect handler_module} (#{inspect self()}) connection error: #{:inet.format_error(reason)} (#{reason})" end)
+        # :ssl.format_error handles posix error as well as ssl
```

Comment:
Should the new `Thrift.ConnectionError.message/1` code use `:ssl.format_error` too?